roboteam_ai/src/conditions/IsBallCloseToBorder.cpp

```diff
@@ -0,0 +1,44 @@
+//
+// Created by robzelluf on 2/18/19.
+//
+
+#include "IsBallCloseToBorder.h"
+
+namespace rtt{
+namespace ai {
+
+IsBallCloseToBorder::IsBallCloseToBorder(std::string name, bt::Blackboard::Ptr blackboard)
+        :Condition(std::move(name), std::move(blackboard)) { };
+
+void IsBallCloseToBorder::initialize() {
+    if (properties->hasDouble("margin")) {
+        margin = properties->getDouble("margin");
+    }
+}
+
+bt::Node::Status IsBallCloseToBorder::update() {
+    Vector2 ballPos = World::getBall()->pos;
+    if (!properties->getBool("corner")) {
```

Comment:
also use hasBool?

Reply:
getBool defaults false

Reply:
I still agree with lukas, this is hard to read